deployment/ansible/roles/driver.app/templates/nginx-app.conf.j2

```diff
@@ -13,7 +13,7 @@ server {
         proxy_set_header X-Forwarded-Server $host;
         proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
 
-        try_files $uri $uri/ /index.html;
```

Comment:
I've tried clicking around everything in the web app, and it seems fine. Not sure if there's maybe some other browsers/configurations that might not work with this.